김민진/src/main/java/org/bank/Main.java

```diff
@@ -0,0 +1,9 @@
+package org.bank;
+
+public class Main {
+    public static void main(String[] args) {
+
+        BankApplication bankApplication = new BankApplication();
+
+    }
+}
```

Comment:
```suggestion
}

```
EOL(End Of Line)에 대해 알고 적용하면 좋을 것 같습니다.

https://velog.io/@bjy991018/EOLEndOfLine